package.json

```diff
@@ -78,6 +78,7 @@
     "ignore": [
       "!lib",
       "src",
+      "test",
```

Comment:
btw, I would prefer to declare what to include instead of what to exclude.. which can be done with the top level [files field](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#files)

```json
"files": [
    "lib",
    "LICENSE",
    "README.md",
    "package.json"
 ],
```

Reply:
The `files` field is dangerous and should never be used; accidentally omitting a necessary file is far worse than accidentally including an unnecessary one.